include/z64actor.h

```diff
@@ -640,4 +677,27 @@ typedef enum {
     /* 0xFF */ TATL_HINT_ID_NONE = 0xFF
 } TatlHintId;
 
+typedef struct {
+    /* 0x0 */ f32 rangeSq;
+    /* 0x4 */ f32 leashScale;
+} TargetRangeParams; // size = 0x8
+
+// From OoT's struct_80034A14_arg1
+typedef struct struct_800BD888_arg1 {
+    /* 0x00 */ s16 unk_00;
+    /* 0x02 */ s16 unk_02;
+    /* 0x04 */ s16 unk_04;
+    /* 0x06 */ s16 unk_06;
+    /* 0x08 */ Vec3s unk_08;
+    /* 0x0E */ Vec3s unk_0E;
+    /* 0x14 */ f32 unk_14;
+    /* 0x18 */ Vec3f unk_18; // Usually setted to Player's position or Player's focus
+    /* 0x24 */ s16 unk_24;
+} struct_800BD888_arg1; // size = 0x28
```

Comment:
Addressed in the NPC tracking PR, so would simply conflicts if this is returned to z64.h, but not a big deal